doc/source/user_guide/missing_data.rst

```diff
@@ -920,3 +920,27 @@ filling missing values beforehand.
 
 A similar situation occurs when using Series or DataFrame objects in ``if``
 statements, see :ref:`gotchas.truth`.
+
+NumPy ufuncs
```

Comment:
there is a section in dsintro.rst that references ufuncs, I would at the very least reference it.